.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
nit: only add files that are meaningful to the PR, you can navigate between your changes with `git add -p` so you can apply only the necessary ones